packages/ui/src/partials/w3m-walletconnect-qr/index.ts

```diff
@@ -46,7 +46,7 @@ export class W3mWalletConnectQr extends LitElement {
     return html`
       <div class="w3m-qr-container">
         ${this.uri
-          ? html`<w3m-qrcode size=${this.overlayEl.offsetWidth} uri=${this.uri}> </w3m-qrcode>`
+          ? html`<w3m-qrcode size="${this.overlayEl.offsetWidth}" uri=${this.uri}></w3m-qrcode>`
```

Comment:
Minor/ No Action: What was the difference to this ?

Reply:
I think this might just be formatting from lit dependency updates